### ISSUE TITLE
Lookup file format from extension

### ISSUE THREAD

**Issue by MOZGIII** [mmalecot/file-format#64]
Can we have the ability to lookup the file format from the file extension?

**Comment by mmalecot** [comment]
Yes, that's planned, there's a PR (#63) I need to review that implements the `from_extension` function.

**Comment by mmalecot** [comment]
Hi,
I'm closing this issue as I'll be working on #63.
